### PR TITLE
Add override-votes on Gerrit Trigger

### PR DESCRIPTION
**Pull request by erenatas**
Added the `override-votes` key on Gerrit Trigger implementation
as it's necessary for vote values.

### CODE REVIEW

File: jenkins_job_wrecker/modules/triggers.py [hunk 1]
```diff
@@ -238,6 +243,14 @@ def gerrittrigger(top, parent):
             pass  # Unconfigurable Attribute
         else:
             raise NotImplementedError("Not implemented Gerrit Trigger Plugin's attribute: ", child.tag)
+
+        if child.tag in mapper_gerrit_build.get_tags():
+            # Jenkins Job Builder implementation uses "override-votes"
+            # key to override default vote values. For detail:
+            # https://docs.openstack.org/infra/jenkins-job-builder/triggers.html#triggers.gerrit
+            is_override_votes = True
```
Comment (romanek-adam) [inline jenkins_job_wrecker/modules/triggers.py]: Would it be ok to put this under:

    elif mapper_gerrit_build.map_element(child, gerrit_trigger):

?

Then you wouldn't need `.get_tags()` at all.

Comment (erenatas) [reply jenkins_job_wrecker/modules/triggers.py]: Yes it would, updating the code.

**Review by romanek-adam — APPROVED** [review]

**Comment by romanek-adam** [comment]
The commit message no longer reflects the reality. @erenatas please update it.

**Review by vdanaila — APPROVED** [review]

**Review by romanek-adam — CHANGES_REQUESTED** [review]
Please update the commit message

**Review by romanek-adam — APPROVED** [review]

**Pull request merged**